libs/wingsdk/test/target-tf-gcp/bucket.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as cloud from "../../src/cloud";
+import * as tfgcp from "../../src/target-tf-gcp";
+import { mkdtemp } from "../../src/util";
+import { tfResourcesOf, tfSanitize, treeJsonOf } from "../util";
+
+test("create a bucket", () => {
```

Comment:
let's add a multiple bucket test, since we are adding it with other providers (in PRs) I think it makes sense to add one here as well, just for sanity.

Reply:
Good idea, adding...